tests/providers/amazon/aws/operators/test_eks.py

```diff
@@ -0,0 +1,299 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import unittest
+from typing import List
+from unittest import mock
+
+from moto.eks.responses import DEFAULT_NEXT_TOKEN
+
+from airflow.providers.amazon.aws.hooks.eks import EksHook
+from airflow.providers.amazon.aws.operators.eks import (
+    EKSCreateClusterOperator,
+    EKSCreateNodegroupOperator,
+    EKSDeleteClusterOperator,
+    EKSDeleteNodegroupOperator,
+    EKSDescribeAllClustersOperator,
+    EKSDescribeAllNodegroupsOperator,
+    EKSDescribeClusterOperator,
+    EKSDescribeNodegroupOperator,
+    EKSListClustersOperator,
+    EKSListNodegroupsOperator,
+)
+from tests.providers.amazon.aws.utils.test_eks_constants import (
```

Comment:
```suggestion
from tests.providers.amazon.aws.utils.eks_constants import (
```

If that file doesn't contain any tests we shouldn't prefix it with `test_`

Reply:
Here and below, can we compromise on `eks_test_{foo}.py`?  They don't contain tests, so I see what you are saying, but they are only used by the EKS tests and I think that should be inherent in the name.

Reply:
Renamed in a coming revision.

Reply:
Addressed in https://github.com/apache/airflow/pull/16571/commits/66679787a678bcce6563f9a600a2ccadd3a5d269 before I found out you don't want everything squashed, apologies for the confusion/complication.